packages/expo-cli/src/exp.ts

```diff
@@ -849,10 +847,33 @@ async function writePathAsync() {
   await Binaries.writePathToUserSettingsAsync();
 }
 
+async function bootstrapAnalyticsAsync(): Promise<void> {
+  Analytics.initializeClient('vGu92cdmVaggGA26s3lBX6Y5fILm8SQ7', packageJSON.version);
+  UnifiedAnalytics.initializeClient('u4e9dmCiNpwIZTXuyZPOJE7KjCMowdx5', packageJSON.version);
+
+  const userData = await profileMethod(UserManager.getCachedUserDataAsync)();
+
+  if (!userData?.userId) return;
+
+  UnifiedAnalytics.identifyUser(userData.userId, {
+    userId: userData.userId,
+    currentConnection: userData?.currentConnection,
+    username: userData?.username,
+    userType: '', // not available without hitting api
+  });
+
+  const command = process.argv[2];
```

Comment:
just want to confirm that we can assume argv[2] will always exist at this point in the app? i see `writePathAsync()` accesses it without a try/catch or if statement so I assume it's safe to do this here.